gradle/libs.versions.toml

```diff
@@ -79,6 +79,7 @@ flink = "1.18.0"
 cglib = "2.2"
 ranger = "2.4.0"
 javax-jaxb-api = "2.3.1"
+javax-ws-rs-api = "2.1.1"
```

Comment:
Why do we need to add this library? I don't find there's a place using it.

Reply:
I use it `implementation(libs.javax.ws.rs.api)` in the authorizations/authorization-ranger/build.gradle.kts:L56
Because JDK-17 removed `Javax` library internally, So we need to manually add it.